lib/rucio/daemons/automatix/automatix.py

```diff
@@ -146,25 +148,17 @@ def automatix(inputfile: str, sleep_time: int, once: bool = False) -> None:
 
 
 def run_once(heartbeat_handler: HeartbeatHandler, inputfile: str, **_kwargs) -> bool:
-
     _, _, logger = heartbeat_handler.live()
+
     try:
-        rses = config_get_list("automatix", "rses")
+        rse_expr = config_get("automatix", "rses")
+        rses = sorted([rse['rse'] for rse in parse_expression(rse_expr)])
```

Comment:
Adding a note that we should also update the docs https://rucio.github.io/documentation/operator/configuration_parameters/#automatix ("rses: Separated by commas.") when this change is released

Reply:
Good catch, thank you.